Doc/library/platform.rst

```diff
@@ -359,3 +359,15 @@ Android Platform
    <https://storage.googleapis.com/play_public/supported_devices.html>`__.
 
    .. versionadded:: 3.13
+
+
+Miscellaneous
+-------------
+
+.. function:: invalidate_caches()
+
+   Clear out the internal cache of information, such as the :func:`uname`.
+   This is typically useful when the platform's :func:`node` is changed
+   by an external process and one needs to retrieve the updated value.
+
+   .. versionadded:: 3.14
```

Comment:
I should have used a versionadded:: next here but at that time, it didn't exist yet. Not sure if this warrants an additional commit though. WDYT @vstinner

Reply:
next will be replaced with 3.14 anyway, there is no need to replace it.

Reply:
Yea but sometimes users are confused that the feature is present in the docs but not in the current release. That was the purpose of `next`